dbms/src/DataTypes/DataTypesDecimal.h

```diff
@@ -131,23 +131,14 @@ class DataTypeDecimal final : public DataTypeSimpleSerialization
     bool isParametric() const override { return true; }
     bool haveSubtypes() const override { return false; }
     bool shouldAlignRightInPrettyFormats() const override { return true; }
-    bool textCanContainOnlyValidUTF8() const override { return true; }
```

Comment:
I cannot find the replacement.

textCanContainOnlyValidUTF8 - that is true for decimal.